msal/build.gradle

```diff
@@ -188,12 +188,12 @@ task sourcesJar(type: Jar) {
 
 // In dev, we want to keep the dependencies (common4j, common) to 1.0.+ to be able to be consumed by daily dev pipeline.
 // In release/*, we change these to specific versions being consumed.
-def commonVersion = "20.1.0"
+def commonVersion = "20.1.1-RC3"
```

Comment:
Is the RC tag still supposed to be here?

Reply:
I missed this; thank you. Updated.